README.md

```diff
@@ -112,41 +122,58 @@ the page.
 ### Constructor
 
 ```ts
+
 new KoaWebMonetization(opts: Object | void): KoaWebMonetization
 ```
 
 Create a new `KoaWebMonetization` instance.
 
 - `opts.plugin` - Supply an ILP plugin. Defaults to using Moneyd.
 - `opts.maxBalance` - The maximum balance that can be associated with any user. Defaults to `Infinity`.
-
+- `opts.receiveEndpointUrl` - The endpoint in your Koa route configuration that specifies where a user pays streams PSK packets to your site. Defaults to `/__monetizer/{id}` where `{id}` is the server generated ID (stored in the browser as a cookie).
+- `opts.cookieName` - The cookie key name for your server generated payer ID. Defaults to `__monetizer`.
+- `opts.cookieOptions` - Cookie configurations for Koa. See [Koa ctx setting cookies options](http://koajs.com/) for more details! Only defaults are `httpOnly: false`
 ### Receiver
 
 ```ts
 instance.receiver(): Function
```

Comment:
This doesn't seem right; the example above uses `instance.receive.bind(instance)` instead of `instance.receiver()`